src/main/java/com/mraof/minestuck/item/MSItemGroup.java

```diff
@@ -11,7 +11,8 @@
 	public static final ItemGroup MAIN = new MSItemGroup("minestuck", () -> new ItemStack(MSItems.CLIENT_DISK));
 	public static final ItemGroup LANDS = new MSItemGroup("minestuck_lands", () -> new ItemStack(MSBlocks.GLOWING_MUSHROOM));
 	public static final ItemGroup WEAPONS = new MSItemGroup("minestuck_weapons", () -> new ItemStack(MSItems.ZILLYHOO_HAMMER));
-	
+	public static final ItemGroup DECORATIONS = new MSItemGroup("minestuck_decorations", () -> new ItemStack(MSBlocks.CASSETTE_PLAYER_CLOSED));
```

Comment:
There are rather few items that have been put in this item group. Is it really needed at this time?

Reply:
I can remove it if you want. I created it for two purposes :
1 - Learning how to
2 - As a reminder of the fact we don't have enough decorative blocks

Reply:
In that case, I'd suggest that you remove it